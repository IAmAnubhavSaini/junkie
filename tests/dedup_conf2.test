#!/bin/sh

NAME="Deduplication Configuration"
PCAPS="pcap/eth/various_vlans.pcap"
filter() {
	grep '^Ethernet'
}
# As we do not collapse vlans anymore then we must see the three packets
CMDLINE="(set-collapse-vlans #f)"
OUTPUT="dedup_conf2.out"

. "$srcdir/dump_test"
