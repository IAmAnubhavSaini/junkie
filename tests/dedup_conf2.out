-- pcap/eth/various_vlans.pcap --
Ethernet: head_len=18, payload=151, vlan_id=1, source=12:34:56:78:90:ab, dest=ab:09:87:65:43:21, proto=IPv4
Ethernet: head_len=18, payload=151, vlan_id=2, source=12:34:56:78:90:ab, dest=ab:09:87:65:43:21, proto=IPv4
Ethernet: head_len=14, payload=151, vlan_id=-1, source=12:34:56:78:90:ab, dest=ab:09:87:65:43:21, proto=IPv4
